CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2992,6 +2992,7 @@ end
 
 local function getDeveloperConsole()
 	if not myDeveloperConsole then
+		myDeveloperConsole = true -- debounce
```

Comment:
We're going to use another variable instead of myDeveloperConsole for the debounce.
Your fix will be a part of: https://github.com/ROBLOX/Core-Scripts/pull/540